modules/fb_instant_articles_display/src/DrupalInstantArticleDisplay.php

```diff
@@ -28,8 +30,11 @@
 /**
  * Facebook Instant Article node wrapper class.  Builds up an InstantArticle
  * object using field formatters.
+ *
+ * Class DrupalInstantArticleDisplay
+ * @package Drupal\fb_instant_articles_display
  */
-class InstantArticleNodeWrapper {
+class DrupalInstantArticleDisplay {
```

Comment:
Derp :/  sorry about that @simonengelhardt.  Didn't see your comments come in on the PR until after I merged it.